articles/client-platforms/react/07-authorization.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Authorization
+description: This tutorial will show you how assign roles to your users, and use those claims to authorize or deny a user to access certain routes in the app.
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-samples/auth0-react-sample/tree/master/07-Authorization',
+}) %>_
+
+<%= include('../_includes/_authorization-introduction', { ruleslink: '/quickstart/spa/react/06-rules' }) %>_
+
+## 2. Check if user's role is present
```

Comment:
@flaviogranero There is no `1.` in the doc. I assume this should be it `07-authorization#create-a-rule-to-assign-roles`. Please update.

Also, the `#create-a-rule-to-assign-roles` is H3 (`### Create a Rule to assign roles`), while the rest in this doc are H2. This needs to be aligned.